src/cli.rs

```diff
@@ -24,6 +25,7 @@ impl Selector {
             if desc.len() > 0 {
                 desc = format!(" - {}", desc);
             }
+            #[cfg(not(target_os = "windows"))]
```

Comment:
It would be good to add a fallback in the Windows case that actually prints the same line, just without the colors, otherwise Windows users won't see anything.

Reply:
Done. I've also changed to the target_family configuration instead of target_os as it's much cleaner to check for families rather than specific os.